routes/api/v1/items.js

```diff
@@ -0,0 +1,25 @@
+const express = require( 'express' )
+const router = express.Router()
+const itemUpdate = require('../../items/item_update')
+
+const findAllItems = require('../../items/find_all_items')
+
+router.get( '/', ( request, response ) => {
+  const { Item } = request.app.get( 'models' )
+  const { query, decoded } = request
+
+  findAllItems( Item, decoded.user, query )
+    .then( data => response.json( { data } ) )
+})
+
+router.post( '/create', ( request, response ) => {
```

Comment:
Let's make this consistent with the existing routes, and remove the `create` portion of the url - a simple post to `/items` with the necessary data is sufficient.